numba/testing/main.py

```diff
@@ -399,9 +399,9 @@ def _choose_gitdiff_tests(tests, *, use_common_ancestor=False):
     path = os.path.join('numba', 'tests')
     if use_common_ancestor:
         print(f"Git diff by common ancestor")
-        target = 'origin/master...HEAD'
+        target = 'origin/release0.55...HEAD'
     else:
-        target = 'origin/master..HEAD'
+        target = 'origin/release0.55..HEAD'
```

Comment:
Am wondering if this should be done on every release branch as part of the release process.

Reply:
probably a good idea